katdal/__init__.py

```diff
@@ -50,7 +50,7 @@ def filter(self, record):
     import katversion as _katversion
 except ImportError:
     import time as _time
-    __version__ = "0.0+unknown.%s" % (_time.strftime('%Y%m%d%H%M'),)
+    __version__ = "0.0+unknown.{}".format(_time.strftime('%Y%m%d%H%M'))
```

Comment:
Already done 5 years ago, it seems 😂 This code is only in the `katversion` docs, though, since it doesn't really care what's in between the VERSION CHECK barriers.

Reply:
Already using `.format` 5 years ago, it seems :joy: It's only in the `katversion` docs, too, since the code itself don't care what's between the `VERSION CHECK` barriers.